### PR TITLE
docs: terminal doc update to include local filesystem sync

### PR DESCRIPTION
**Pull request by danielb2**

### CODE REVIEW

File: readme/terminal.md [hunk 1]
```diff
@@ -135,6 +135,13 @@ When syncing with Dropbox, Joplin creates a sub-directory in Dropbox, in `/Apps/
 
 To initiate the synchronisation process, type `:sync`. You will be asked to follow a link to authorise the application.
 
+## Local filesystem synchronisation
+
+Local filesystem sync can be initiatd after starting the joplin terminal app by using the [command-line mode](#command-line-mode)
```
Comment (tessus) [inline readme/terminal.md:140]: small typo.... it's `initiated`

**Review by tessus — CHANGES_REQUESTED** [review]
Please see my inline comment

**Comment by tessus** [comment]
Not entirely sure, if this addition is really needed because it is described in the `Commands` section, but it also does not hurt. For consistency it might be good to add it....

**Comment by danielb2** [comment]
> Not entirely sure, if this addition is really needed because it is described in the `Commands` section, but it also does not hurt. For consistency it might be good to add it....

commands sections are examples with "possible values" and aren't documentation as such IMO. Either way, I didn't find it when I was looking because other references  talk about the `filesystem` so that's what I searched for. I also read the Synchronisation and saw all the other options, but not filesystem. Makes sense to me to add it.

**Review by tessus — APPROVED** [review]
Thanks for the fix.

**Pull request merged**